### ISSUE TITLE
About  the 'Enable Readability' option of Actions's side effect

### ISSUE THREAD

**Issue by anaughtyboy** [RSSNext/Follow#797]
### Describe the bug

First, I switched the  'Enable Readability' on, which is one of the Actions's options. Second, I opened an acticle, then something weird happened, the content area got stuck and was replaced with some words like "Fetching original content and processing..."

### Feed Info

my feed url: https://app.follow.is/feeds/all/41223695307544718?view=0
source: https://diygod.cc/6-year-of-rsshub

### Reproduction Video

![step1](https://github.com/user-attachments/assets/bc9e0813-9539-48a4-ab65-6f899f9c1398)
![step2](https://github.com/user-attachments/assets/d627fc17-3534-49ec-9e75-31a1934e2c13)

### App Type

Web

### Operating System

macOS

### Browser Info

Chrome 116.0.5845.187 (Official Build) (x86_64)

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A, please open a GitHub Discussion instead.

### Contributions

- [ ] I am willing to submit a PR to fix this issue
- [ ] I am willing to submit a PR with failing tests (actually just go ahead and do it, thanks!)

**Comment by hyoban** [comment]
This may be related to your network environment. Whether you can open the source website(https://diygod.cc/6-year-of-rsshub)?

**Comment by anaughtyboy** [comment]
> This may be related to your network environment. Can you open the source website(https://diygod.cc/6-year-of-rsshub)?
> 
> You can try to set a network proxy in settings.

Yeah,  I can open the source url. The only way to trigger the reloading (showing up "Fetching orign ...") is to switch the  'Enable Readability' on. Otherwise, when switch it off, everything is ok. so  weird.
